public/js/test/mochitest/browser_dbg-editor-select.js

```diff
@@ -6,39 +6,34 @@
 
 const TAB_URL = EXAMPLE_URL + "doc-scripts.html";
 
-function assertPausedLocation(dbg, source, line) {
-  const { selectors: { getSelectedSource, getPause }, getState } = dbg;
-
-  is(getSelectedSource(getState()).get("url"), source.url);
-  const location = getPause(getState()).getIn(["frame", "location"]);
-  is(location.get("sourceId"), source.id);
-  is(location.get("line"), line);
+// checks to see if the first breakpoint is visible
+function isBreakpointVisible(dbg) {
+  const bpLine = findElement(dbg, "breakpoint");
+  const cm = findElement(dbg, "codeMirror");
+  ok(isVisibleWithin(cm, bpLine), "CodeMirror is scrolled to line");
+}
 
-  ok(dbg.win.cm.lineInfo(line - 1).wrapClass.includes("debug-line"),
-     "Line is highlighted");
+function assertNoSelectedSource(dbg) {
+  const { selectors: { getSelectedSource }, getState } = dbg;
+  ok(!getSelectedSource(getState()), "No selected source");
 }
 
 add_task(function* () {
   const dbg = yield initDebugger(
     TAB_URL,
     "simple1.js", "simple2.js", "long.js"
   );
-  const { selectors: { getSelectedSource }, getState } = dbg;
   const simple1 = findSource(dbg, "simple1.js");
   const simple2 = findSource(dbg, "simple2.js");
 
   // Set the initial breakpoint.
   yield addBreakpoint(dbg, simple1.id, 4);
-  ok(!getSelectedSource(getState()), "No selected source");
+  assertNoSelectedSource(dbg);
```

Comment:
When possible, I'd like to just query the state directly. Creating helpers is good when you have to do a bunch of extra work, but otherwise it forces you to jump around to see what it's actually asserting. Do you mind reverting this change?

One thing that helps is we can probably stop labeling every single assert. It's good in some places, but we can just do `ok(!getSelectedSource(getState()))`. No other testing framework makes you label every single assert, and if it fails it tells you the line number of the failure.